e2e/playwright/info-drawer/src/tests/comments.spec.ts

```diff
@@ -60,6 +60,7 @@ test.describe('Info Drawer - Comments', () => {
     await nodesApi.createFolder(personalFolderName);
     await fileActionsApi.waitForNodes(personalFolderName, { expect: 1 });
     await personalFiles.navigate();
+    await Utils.reloadPageIfRowNotVisible(personalFiles, personalFolderName);
```

Comment:
Could you please explain why that function is added to so many tests? Was not that just for example environmental bug or other temporary issue which prevents displaying row?

Reply:
It seems like "await fileActionsApi.waitForNodes(personalFolderName, { expect: 1 });" is not enough to wait to load test nodes, so tests opened an empty dataTable on "await personalFiles.navigate();". Thats why I added this Utils method so it refreshes the page if the dataTable is empty OR the test node is not visible.

It dont know where the issue was exactly, so I came up with this fix. It seems like this was not an issue in Playwright 1.37.1 so MAYBE the 1.42.1 broke the waitForNodes method, but I am not sure, because it still works, just the tests are not loading nodes when navigating on the page.

Reply:
ok thank you for explanation